parser_library/include/workspace_manager.h

```diff
@@ -130,7 +130,11 @@ class PARSER_LIBRARY_EXPORT workspace_manager
 private:
     impl* impl_;
 };
-
+struct asm_option
```

Comment:
this struct should be defined somewhere else... Workspace manager contains the interface between `parser_library` and `language_server`.
It should probably be defined in its separate file inside `hlasm_plugin::parser_library` namespace (so in `parser_library/src/` folder)